src/content/docs/en/reference/feature-flags/legacy-collections.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Legacy - Collections
```

Comment:
The hyphen is weird here to me.

Should it be one of these?

> Legacy: Collections

> Legacy collections

Reply:
Yes, canonically I think we have preferred colons if using punctuation in the past, so def. `:` if punctuation ends up being used.